ide/jumpto/src/org/netbeans/modules/jumpto/type/GoToTypeAction.java

```diff
@@ -498,12 +495,13 @@ public void run() {
                     caseSensitive,
                     GoToSettings.getDefault().isSortingPreferOpenProjects());
             itemsComparator = ic;
-            final Models.MutableListModel baseModel = Models.mutable(
+            final Models.MutableListModel<TypeDescriptor> baseModel = Models.mutable(
                 ic,
                 currentSearch.resetFilter(),
                 null);
-            final ListModel model = typeFilter != null ?
-                    FilteredListModel.create(baseModel, new FilterAdaptor(typeFilter), NbBundle.getMessage(GoToTypeAction.class, "LBL_Computing")) :
+            @SuppressWarnings("unchecked")
+            final ListModel<TypeDescriptor> model = typeFilter != null ?
+                    FilteredListModel.create(baseModel, new FilterAdaptor(typeFilter), null) :
                     baseModel;
```

Comment:
this was what passed a String into the model + replaced the typed model with it a few lines later. The initial value is now null since it wouldn't show up anyway.